src/main/java/domain/Count.java

```diff
@@ -0,0 +1,28 @@
+package domain;
+
+public class Count {
+
+    private static final int MIN_COUNT = 0;
+    private static final int MAX_COUNT = 100;
+
+    private int count;
+
+    public Count(int count) {
+        validateCount(count);
+        this.count = count;
+    }
+
+    private void validateCount(int count) {
+        if (count <= MIN_COUNT || count > MAX_COUNT) {
+            throw new IllegalArgumentException();
+        }
+    }
+
+    public boolean hasCount() {
+        return count > 0;
```

Comment:
```suggestion
        return count > MIN_COUNT;
```
만들어주신 상수를 사용할 수 있겠네요 😄